sdl_android/src/main/java/com/smartdevicelink/proxy/rpc/ClimateControlData.java

```diff
@@ -3,104 +3,181 @@
 import com.smartdevicelink.proxy.RPCStruct;
 import com.smartdevicelink.proxy.rpc.enums.DefrostZone;
 import com.smartdevicelink.proxy.rpc.enums.VentilationMode;
+
 import java.util.Hashtable;
 
-public class ClimateControlData extends RPCStruct{
```

Comment:
For history, is it possible to only change the additions and not the whole class? It also makes it easier to review.